### ISSUE TITLE
Type error for disabled button state

### ISSUE THREAD

**Issue by bk3c** [bloom-housing/bloom#587]
Looks like I let this sneak in when merging #561:

```
Failed to compile.

./pages/applications/review/terms.tsx:97:17
Type error: Type '{ children: string; disabled: true; filled: true; onClick: () => void; }' is not assignable to type 'IntrinsicAttributes & ButtonProps'.
  Property 'disabled' does not exist on type 'IntrinsicAttributes & ButtonProps'.

   95 |             <div className="form-card__pager-row primary">
   96 |               <Button
>  97 |                 disabled
      |                 ^
   98 |                 filled={true}
   99 |                 onClick={() => {
  100 | 
```

**Comment by dominikx96** [comment]
@bk3c should be fixed

**Comment by bk3c** [comment]
Fixed in #593
